packages/open-next/src/build.ts

```diff
@@ -95,7 +95,7 @@ export async function build(
   await createRevalidationBundle(config);
   await createImageOptimizationBundle(config);
   await createWarmerBundle(config);
-  await generateOutput(options.appPath, options.appBuildOutputPath, config);
+  await generateOutput(options.appBuildOutputPath, config);
```

Comment:
Thanks, LGTM.
As a side note for a new Issue ticket for potential build performance:
Can we wrap all the `create...Bundle` functions on line 94-97 in a `Promise.all` to run the bundling in parallel?
I believe they should all be independent and don't depend on each other.
I'm not sure if the time save would be noticeable though.

Reply:
It would probably not make a very big difference, the only 2 that takes a bit of time are the imageOptimization (because of sharp install) and the serverFunction themselves, but yeah that something that we could add.